bin/generate_schema_for_component.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Run this script from the root of the repository
+
+set -e
```

Comment:
Let's also activate the OAS check in gh actions: https://github.com/maykinmedia/open-vtb/blob/main/.github/workflows/oas.yml#L1